packages/package_info/example/android/app/src/main/java/io/flutter/plugins/packageinfoexample/MainActivity.java

```diff
@@ -4,14 +4,15 @@
 
 package io.flutter.plugins.packageinfoexample;
 
-import android.os.Bundle;
-import io.flutter.app.FlutterActivity;
-import io.flutter.plugins.GeneratedPluginRegistrant;
+import dev.flutter.plugins.e2e.E2EPlugin;
+import io.flutter.plugins.packageinfo.PackageInfoPlugin;
+import io.flutter.embedding.android.FlutterActivity;
+import io.flutter.embedding.engine.FlutterEngine;
 
 public class MainActivity extends FlutterActivity {
   @Override
-  protected void onCreate(Bundle savedInstanceState) {
-    super.onCreate(savedInstanceState);
-    GeneratedPluginRegistrant.registerWith(this);
+  public void configureFlutterEngine(FlutterEngine flutterEngine) {
+    flutterEngine.getPlugins().add(new PackageInfoPlugin());
```

Comment:
nit: add a TODO to remove this once the generated registrant for the v2 embedder is on stable

Reply:
done, we should probably make this change in the battery plugin also

https://github.com/flutter/plugins/blob/master/packages/battery/example/android/app/src/main/java/io/flutter/plugins/batteryexample/MainActivity.java